package.json

```diff
@@ -149,7 +154,9 @@
     "style-loader": "^3.2.1",
     "ts-node": "^10.0.0",
     "tsconfig-paths": "^3.9.0",
-    "typescript": "^4.5.4"
+    "typescript": "^4.5.4",
+    "webpack": "^5.68.0",
+    "webpack-cli": "^4.9.2"
```

Comment:
We could create a `package.json` specifically for tests, instead of pollute a global one.